File: 0035-search-insert-position/0035-search-insert-position.java
class Solution {
    public int searchInsert(int[] nums, int target) {
        
        // finding the index if target is present inside the nums
        for( int i = 0; i < nums.length; i++){
            if (nums[i] == target){
                return i;
            }
        }

        // finding a index where target shuold be placed
        for( int i = 0; i < nums.length; i++){
            if (nums[i] > target){
                return i;
            }
        }
        
        // target should be placed at the end 
        return nums.length;
    }
}

File: 0035-search-insert-position/0035-search-insert-position.py
class Solution:
    def searchInsert(self, nums: List[int], target: int) -> int:

        # finding the index if target is present inside the nums
        for number in nums:
            if number == target:
                return nums.index(number)
            
        # finding a index where target shuold be placed
        for number in nums:
            if number > target:
                return nums.index(number)

        # target should be placed at the end   
        return len(nums)
        